addons/hr_holidays/views/hr_leave_type_views.xml

```diff
@@ -49,8 +49,7 @@
                             <field name="time_type" groups="base.group_no_one"/>
                         </group>
                         <group name="validation" string="Validation">
-                            <field name="validation_type" widget="radio" invisible="1"/>
-                            <field name="double_validation"/>
+                            <field name="validation_type" widget="radio"/>
```

Comment:
I wasn't sure if removing a field from a view was stable or not, I'll make it invisible instead of removing it

Reply:
Well even if you make it invisible in the view, it will cause issue since the field doesn't exist anymore, people will have to do a module upgrade to avoid having issues.

But as I said, don't bother yourself we are the only one using this version, so we will pay attention when deploying it.